packages/client/src/Components/Modals/AddRoleModal.jsx

```diff
@@ -131,6 +131,8 @@ class AddRoleModal extends Component {
                 options={{numInput: false}}
               />
             </div>
+            {selectedForce === UMPIRE_FORCE &&
```

Comment:
Note: the code won't currently build, since `selectedForce` isn't visible in this scope.